pages/micromaster/profiles/roshini75.md

```diff
@@ -0,0 +1,19 @@
+## Details:-
+* Name  -  **Roshini Tadi**
+* location/time zone - **Iam a student at UTA, Arlington, Texas, time zone - CST**
+* Operating System - **Windows 10 Home**
+
+## About me
+
+I'm passionate about learning the latest technologies related to both frontend and backend. I really enjoy creating responsive and user-friendly applications. I would like to work in fast-paced environments with high educational and cultural diversity. I have two years of experience in both frontend and backend technologies. I am a pragmatic thinker and have a great predilection towards problem-solving. I'm an adaptable and self-motivated learner. Interested in new programming technologies and continuous self-improvement. I'm currently open to internship opportunities for fall 2020 within the software engineering discipline.
+You can contact me at
+
+My college email [Id](roshini.tadi@mavs.uta.edu)
```

Comment:
if you check rawgithack, this email link is not accurate and it does not open email agent.